projects/Rockchip/devices/RK3326/linux/linux.aarch64.conf

```diff
@@ -1,20 +1,21 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/arm64 6.7.2 Kernel Configuration
+# Linux/arm64 6.8.9 Kernel Configuration
```

Comment:
Kernel config update? Might explain some of the seemingly unrelated changes here.